### DISCUSSION TITLE
Nativewind not working in expo

### DISCUSSION THREAD

**Discussion by grandwaterman** [nativewind/nativewind#555] [Q&A]
Followed the doc https://www.nativewind.dev/quick-starts/expo. 

The three steps, and it is NOT working. and checking the example on Github, it is very different from the doc.

**Got error: Use process(css).then(cb) to work with async plugins**

package.json - dep
  "dependencies": {
    "autoprefixer": "^10.4.15",
    "expo": "~49.0.10",
    "expo-status-bar": "~1.6.0",
    "nativewind": "^2.0.11",
    "postcss": "^8.4.29",
    "react": "18.2.0",
    "react-native": "0.72.4",
    "tailwindcss": "^3.3.3"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0"
  },
  
  Document quality -1

**Comment by redkean** [comment]
Use process(css).then(cb) to work with async plugins

is caused by tailwindcss v3.3.3, you should downgrade tailwindcss.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
